### DISCUSSION TITLE
[Script request] Kopia an open-source backup/restore tool

### DISCUSSION THREAD

**Discussion by mieslies** [community-scripts/ProxmoxVE#435] [Request script] [answered]
### Application Name

Kopia server (headless) with GUI and Autostart

### Website

https://github.com/kopia/kopia/

### Description

Kopia is a fast and secure open-source backup/restore tool that allows you to create [encrypted](https://kopia.io/docs/features/#end-to-end-zero-knowledge-encryption) snapshots of your data and save the snapshots to [remote or cloud storage](https://kopia.io/docs/features/#save-snapshots-to-cloud-network-or-local-storage) of your choice, [to network-attached storage or server](https://kopia.io/docs/features/#save-snapshots-to-cloud-network-or-local-storage), or [locally on your machine](https://kopia.io/docs/features/#save-snapshots-to-cloud-network-or-local-storage).  

Kopia does not 'image' your whole machine. Rather, Kopia allows you to backup/restore any and all files/directories that you deem are important or critical. You can run in local, headless and with a GUI or CLI. Kopia is cross-platform for Windows, macOS & Linux 

This request is for headless server (LXC) with GUI in browser and autostart. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
What do you need for this? What is the script in an isolated LXC good for? What is the added value for users?

**Reply by mieslies** [reply]
I have installed ProxMox backup server recently and that script is already available, this will do the job for backup my data. So please cancel my request. Thanks for the reply and your work on the scripts!
